src/adc_readout/PeakFinder.cpp

```diff
@@ -20,17 +20,20 @@ void PeakFinder::processData(SamplingRun const &Data) {
   std::uint64_t PeakTimeStamp = PulseInfo.PeakTimestamp.getTimeStampNS();
 
   if (Serialisers.find(Data.Identifier) == Serialisers.end()) {
-    Serialisers[Data.Identifier] = std::make_unique<RefFilteredEventSerializer>(
+    Serialisers[Data.Identifier] = std::make_unique<EventSerializer>(
         Name + "_Adc" + std::to_string(Data.Identifier.SourceID) + "_Ch" +
             std::to_string(Data.Identifier.ChannelNr),
-        100, 50ms, ProducerPtr.get(),
-        EventSerializer::TimestampMode::INDEPENDENT_EVENTS);
+        200, 500ms, ProducerPtr.get(),
+        EventSerializer::TimestampMode::TIME_REFERENCED);
   }
   auto &CurrentSerialiser = Serialisers[Data.Identifier];
   CurrentSerialiser->addReferenceTimestamp(
       Data.ReferenceTimestamp.getTimeStampNS());
+  auto EventId =
```

Comment:
I'n not sure what happens here. Seems like you're calculating a pixelid outside the 
defined geometry range? If the 512x512 here are related to essgeometry
then you should use essgeometry.nx() and essgeometry.ny() instead.

Reply:
This particular detector does not have any geometry. It is a one pixel monitor. The event-id you see here is based on a request (demand?) from @matthew-d-jones.